utils/web_scraper.py

```
@@ -18,9 +18,54 @@
+# ==================================================
+def fix_missing_bilibili_api_json():
+    """
+    修复缺失的 bilibili_api 数据文件
+
+    当打包后的环境中缺少 bilibili_api 数据文件时，尝试修复该问题。
+    该函数会检查是否存在 bilibili_api 包，并根据环境判断是否需要回退到默认路径。
+    """
+    try:
+        # 优先判断是否是打包后环境
+        if hasattr(sys, '_MEIPASS'):
+            pkg_path = Path(sys._MEIPASS) / "bilibili_api"
```

Ruff (0.15.2)

[error] Local variable `pkg_path` is assigned to but never used

Remove assignment to unused variable `pkg_path`

(F841)